package.json

```diff
@@ -41,6 +41,9 @@
           },
           {
             "filenamePattern": "object*/**/*.{DTAARA,dtaara}"
+          },
+          {
+            "filenamePattern": "object*/**/*.{PGM,pgm,SRVPGM,srvpgm}"
```

Comment:
`"filenamePattern": "object*/**/*.{PGM,SRVPGM}"` is enough, the pattern is not case sensitive